src/NuGet.Core/NuGet.Protocol/LegacyFeed/V2FeedQueryBuilder.cs

```diff
@@ -119,14 +122,7 @@ public class V2FeedQueryBuilder
 
             var orderBy = BuildOrderBy(filters.OrderBy);
 
-            var filter = BuildPropertyFilter(filters.Filter);
-            if (filter != null)
-            {
-                filter = string.Format(
-                    CultureInfo.InvariantCulture,
-                    FilterFormat,
-                    filter);
-            }
+            var filter = BuildFilter(searchTerm, filters, false);
```

Comment:
Just to make it a bit clearer.

```suggestion
            var filter = BuildFilter(searchTerm, filters, includeSearchClauses: false);
```